RELEASING.md

```diff
@@ -0,0 +1,8 @@
+# Sceptre release process
+
+1. Bump version in __init__.py & setup.cfg
+2. Update Changelog
```

Comment:
Probably more specific instructions for what to include could be very useful.

Reply:
Done.